Tribler/Core/plyveladapter.py

```diff
@@ -0,0 +1,43 @@
+import plyvel
+
+class LevelDB(object):
+
+    def __init__(self, store_dir, create_if_missing=True):
+        self._db = plyvel.DB(store_dir, create_if_missing=create_if_missing)
+
+    def Get(self, key, verify_checksums=False, fill_cache=True):
+        return self._db.get(key, verify_checksums=verify_checksums, fill_cache=fill_cache)
+
+    def Put(self, key, value, sync=False):
+        self._db.put(key, value, sync)
+
+    def Delete(self, key, sync=False):
+        return self._db.delete(key, sync)
+
+    def RangeIter(self, key_from=None, key_to=None, include_value=True, verify_checksums=False, fill_cache=True):
```

Comment:
I'd rather use *argv, **kwargs than repeating the if None stuff from the torrent store class.